SF Finder/AppDelegate.swift

```diff
@@ -11,10 +11,10 @@ import SwiftUI
 @main
 class AppDelegate: NSObject, NSApplicationDelegate {
 
+    #warning("TODO: Swap NSPopover implementation with NSView so we get that \"control center\" effect ")
```

Comment:
re this: I think the look we should strive for is the "Control Center" look, i.e. this:

<img width="329" alt="image" src="https://user-images.githubusercontent.com/5796060/108625671-ea1ab780-742a-11eb-8772-9b01fe7dcff2.png">